Gemfile

```diff
@@ -146,7 +146,10 @@ def override_gem(name, *args)
     raise "Trying to override unknown gem #{name}" unless (dependency = dependencies.find { |d| d.name == name })
     dependencies.delete(dependency)
 
-    gem name, *args
+    calling_file = caller_locations.detect {|loc| !loc.path.include?("lib/bundler")}.path
```

Comment:
I know we've discussed this a couple times and decided on spaces for blocks and no spaces for hashes.